### PR TITLE
Add altmetric badges to paper

### PR DESCRIPTION
**Pull request by dominikstrb**
[Altmetric](https://www.altmetric.com/) tracks social media and citation activity for a paper. I added an option to add the altmeric id to a paper's bib entry, which results in the altmeric badge for the paper being displayed next to the other buttons. As an example, I added the altmetric id for the Einstein, Podolsky & Rosen paper.

### CODE REVIEW

File: _layouts/bib.html [hunk 1]
```diff
@@ -177,6 +177,9 @@
           {%- if entry.website %}
             <a href="{{ entry.website }}" class="btn btn-sm z-depth-0" role="button">Website</a>
           {%- endif %}
+          {%- if entry.altmetric %}
+            <script type="text/javascript" src="https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js"></script><div data-altmetric-id="{{ entry.altmetric }}" class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-popover="right" data-badge-type="1" style="display:inline"></div>
```
Comment (alshedivat) [inline _layouts/bib.html:181]: let's change the badge type to 2
```suggestion
            <script type="text/javascript" src="https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js"></script><div data-altmetric-id="{{ entry.altmetric }}" class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-popover="right" data-badge-type="2" style="display:inline"></div>
```

**Review by alshedivat — APPROVED** [review]
LGTM. thanks for adding this feature!

please switch the badge type to "2" and I'll go ahead and merge after that.

**Comment by dominikstrb** [comment]
Alright, I changed the badge type.

Should I add some documentation? If so, where?

**Comment by rohandebsarkar** [comment]
> Should I add some documentation? If so, where?

You can add a description about the altmetric badge along with the `bibtex_keywords` in the `README.md` file (somewhere around line 424).
https://github.com/alshedivat/al-folio/blob/232ff286b22ad2fed3399f033f940f69e6a080cc/README.md?plain=1#L422-L424

**Comment by dominikstrb** [comment]
Thank you!

**Pull request merged**